src/Microsoft.FeatureManagement/ConfigurationFields.cs

```diff
@@ -37,5 +36,10 @@ internal static class ConfigurationFields
         // Other keywords
         public const string NameKeyword = "Name";
         public const string FeatureManagementSectionName = "FeatureManagement";
+
+        // Telemetry keywords
+        public const string Telemetry = "Telemetry";
+        public const string Enabled = "Enabled";
```

Comment:
I suggest "TelemetryEnabled" and ”TelemetryMetadata" for two reasons:
1. We had "FeatureStatus" rather than "Status" in Enum keywords
2. It makes the following code more readable
``` C#
telemetryEnabled = telemetrySection.GetValue<bool>(ConfigurationFields.Enabled);
IConfigurationSection telemetryMetadataSection = telemetrySection.GetSection(ConfigurationFields.Metadata);
```

Reply:
I'd stay away from this pattern. That's setting us up to keep an entire object model hierarchy as part of the name of a field. The name of the fields are enabled and metadata. If other parts of the configuration of a flag can use it they should. Ideally we shouldn't have two different keywords for the concept of Enabled and Metadata.